src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -166,12 +169,13 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
                 paramsMap.put("qdmMatCqlConversionErrors", qdmMatCqlConversionErrors);
                 paramsMap.put("fhirCqlConversionErrors", fhirCqlConversionErrors);
                 paramsMap.put("fhirMatCqlConversionErrors", fhirMatCqlConversionErrors);
-
             }
 
-            Instant instant = Instant.parse(conversionResultDto.getModified());
-            paramsMap.put("runDate", DateUtility.formatInstant(instant, DATE_FORMAT));
-            paramsMap.put("runTime", DateUtility.formatInstant(instant, TIME_FORMAT));
+            if (conversionResultDto.getModified() != null) {
+                Instant instant = Instant.parse(conversionResultDto.getModified());
+                paramsMap.put("runDate", DateUtility.formatInstant(instant, DATE_FORMAT));
```

Comment:
Does this work?
I think you will have to modify validation_report_header.ftl

${runDate} -> ${runDate!""}
${runTime} -> ${runTime!""}